Dockerfile

```diff
@@ -1,43 +1,68 @@
-FROM golang:1.13.5-alpine3.10
+FROM golang:1.14-buster
 
-RUN apk add --no-cache \
-    bash \
+# Remove the default $PS1 manipulation
+RUN rm /etc/bash.bashrc
+
+RUN apt-get update \
+ && apt-get -y --no-install-recommends install \
     bash-completion \
     bats \
-    coreutils \
-    curl \
-    git \
+    gettext \
     groff \
     jq \
     less \
-    openssh-client \
     python3 \
-    ncurses \
-    vim
+    python3-pip
+# ncurses-bin or libncurses6 ?
 
-WORKDIR /root/aladdin
-COPY ./commands/python/requirements.txt ./commands/python/requirements.txt
-RUN pip3 install --no-cache-dir -r ./commands/python/requirements.txt
+# Default to python3 and update setuptools
+RUN ln -fs /usr/bin/python3 /usr/bin/python \
```

Comment:
Why?

Reply:
also might be slightly more correct to use update-alternatives if we want to do this

Reply:
I've moved these symlinks to `/usr/local/bin` so as not to interfere with any os distribution assumptions about which tools will be used.